packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -433,10 +436,10 @@ describe('KeyringController', () => {
       it('should get correct keyring', async () => {
         await withController(async ({ controller }) => {
           const normalizedInitialAccounts =
-            controller.state.keyrings[0].accounts.map(normalize) as string[];
-          const keyring = (await controller.getKeyringForAccount(
-            normalizedInitialAccounts[0],
-          )) as KeyringObject;
+            controller.state.keyrings[0].accounts.map(normalize);
+          const keyring = await controller.getKeyringForAccount(
+            normalizedInitialAccounts[0] as string,
+          );
```

Comment:
Is TypeScript saying that this could be `undefined`? If that's the case, maybe this would be better communicated using a non-null assertion? i.e.

``` suggestion
          const keyring = await controller.getKeyringForAccount(
            normalizedInitialAccounts[0]!,
          );